src/pages/Dashboard/Markets/SupplyMarket/index.tsx

```diff
@@ -99,7 +101,7 @@ export const SupplyMarketUi: React.FC<ISupplyMarketUiProps> = ({
   return (
     <div className={className} css={styles.tableContainer}>
       <Table
-        title={t('markets.supplyMarketTableTitle')}
+        title={isXlDown ? undefined : t('markets.supplyMarketTableTitle')}
```

Comment:
same as other comment. It wasn't possible to do this using styles?

Reply:
the reason of doing this with js described in first comment

Reply:
Because our `Table` component has the title embedded in it, Dmitriy is right that we unfortunately have to use JS for responsiveness.
An alternative approach would be to render the title outside of the table.